src/components/security/SecurityScansTab.tsx

```diff
@@ -218,7 +218,7 @@ export class SecurityScansTab extends Component<RouteComponentProps<{}>, Securit
           {this.state.filtersApplied[filter].map((cluster) => {
             count++;
             return <div key={cluster.value} className="saved-filter">{cluster.label}
-              <button type="button" className="saved-filter__clear-btn pt-4 pb-4"
+              <button type="button" className="dc__saved-filter__close-btn pt-4 pb-4"
```

Comment:
It should be `clear-btn` button, right?

Reply:
let's keep it close for now it is unique. As `saved-filter__clear-btn` we have this as well in base.  Later on will replace close with clear

Reply:
Ok